docs/Migrating.md

```diff
@@ -13,10 +19,13 @@
      in the `Brain` GameObjects
   *  Agents have a `Brain` field in the Inspector, you need to drag the
   appropriate Brain asset in it.
-  
-__Note:__ You can pass the same brain to multiple agents in a scene by 
-leveraging Unity's prefab system or look for all the agents in a scene
-using the search bar of the `Hierarchy` window with the word `Agent`.
+* We have changed the way ML-Agents models perform inference. All previous `.bytes`
+  files can no longer be used. The training process and the shipped models
+  have now a `.tf` extension and use TensorflowSharp as a backend for the 
+  [Inference Engine](Inference-Engine.md).
+  __Note:__ You will need to delete the previous TensorFlowSharp package 
+  and install the new one to do inference.
```

Comment:
Maybe add:

To correctly delete the previous TensorFlowSharp package, Delete all of the files under ML-Agents/Plugins folder except the files under ML-Agents/Plugins/ProtoBuffer.